public/js/components/Graph.js

```diff
@@ -11,29 +11,31 @@ export default class Graph extends React.Component {
     }
 
     componentDidMount() {
-        // TODO: Use actuial data
-        fetch(`keywords/trump`)
-            .then(res => res.json())
-            .then((dataPoints) => {
-                dataPoints = dataPoints
-                    .sort((a, b) => a.times - b.times)
+        fetch(`keywords/top`)
+            .then(res => {
+                return res.json()
+            })
```

Comment:
My code has brackets even if there is only 1 parm being passed. Should we standardize this? @JackyChiu @VikramBombhi @Tuanhao